core/droidkaigiui/src/commonMain/kotlin/io/github/droidkaigi/confsched/droidkaigiui/session/TimetableList.kt

```diff
@@ -62,7 +63,7 @@ fun TimetableList(
             state = lazyListState,
             verticalArrangement = Arrangement.spacedBy(12.dp),
             contentPadding = contentPadding + WindowInsets.navigationBars.asPaddingValues(),
-            modifier = modifier,
+            modifier = modifier.enableMouseDragScroll(lazyListState),
```

Comment:
I didn't want to put a large amount of code unrelated to Android/iOS here, so I separated the processing using expect/actual.